components/brave_sync/brave_sync_service_impl.cc

```diff
@@ -284,7 +284,8 @@ void BraveSyncServiceImpl::BackgroundSyncStarted(bool startup) {
   if (startup)
     bookmark_change_processor_->Start();
 
-  StartLoop();
+  const bool initial_updates = sync_prefs_->GetSyncDevices()->size() <= 1;
```

Comment:
s/initial_updates/waiting_for_second_device/ would be easier to understand why we are doing this

Reply:
thanks, fixed.